tests/configs/test_config_overrides.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+import os
+import unittest
+
+from mmf.utils.configuration import Configuration
+from mmf.utils.env import setup_imports
+from mmf.utils.general import get_mmf_root
+from tests.test_utils import dummy_args
+
+
+class TestConfigOverrides(unittest.TestCase):
```

Comment:
These look good and ready to go. Can you move these to: https://github.com/facebookresearch/mmf/blob/master/tests/utils/test_configuration.py and then we can import it to phabricator.